src/javascript/index.js

```diff
@@ -24,3 +24,4 @@ $(window).on('pageshow', (e) => { // Safari doesn't fire load event when using b
         BinaryLoader.init();
     }
 });
+
```

Comment:
There is an extra empty line in the end, can we remove it?

Reply:
Looks like was added unnecessary changes, removed all, thanks.